src/main/java/org/sonar/plugins/buildbreaker/BuildBreakerPlugin.java

```diff
@@ -83,8 +93,11 @@
 
   public static final String ALTERNATIVE_SERVER_URL_KEY = "sonar.buildbreaker.alternativeServerUrl";
 
+  public static final String FAIL_FOR_ISSUES_WITH_SEVERITY_KEY = "sonar.buildbreaker.preview.failForIssuesWithSeverity";
+  public static final String FAIL_FOR_ISSUES_DISABLED = "DISABLED";
```

Comment:
DISABLED would be a good enough name.  But is it not possible to use a blank string as the disabled option instead?